cron/workspaceHostTransitions.sql

```diff
@@ -56,14 +56,28 @@ WITH
       AND wh.state != 'launching'
     RETURNING
       wh.id
+  ),
+  terminated_workspaces AS (
+    UPDATE workspaces AS w
+    SET
+      state = 'stopped',
+      stopped_at = NOW(),
+      state_updated_at = NOW()
+    FROM
+      terminated_workspace_hosts AS twh
+    WHERE
+      twh.id = w.workspace_host_id
+      AND w.state != 'stopped'
+    RETURNING
+      w.*
   )
-UPDATE workspaces AS w
-SET
-  state = 'stopped',
-  stopped_at = NOW(),
-  state_updated_at = NOW()
+INSERT INTO
+  workspace_logs (date, workspace_id, version, state, message)
+SELECT
+  now(),
```

Comment:
Not that it matters, but all of our `_logs` tables have `DEFAULT now()` for the `date` so we don't have to actually set it.

Reply:
Resolved in 0ecabcc.